registry-manager-service-impl/src/main/java/de/adorsys/registry/manager/service/impl/AspspCsvServiceImpl.java

```diff
@@ -132,21 +132,8 @@ private AspspPO copyContent(AspspPO from) {
     }
 
     private String toCsvString(AspspCsvRecord aspsp) {
-        CsvMapper mapper = new CsvMapper();
-        CsvSchema schema = mapper.schemaFor(AspspCsvRecord.class).withoutQuoteChar();
-
-        if (aspsp.getAspspName() == null) {
```

Comment:
why was this functionality removed?

Reply:
It's not removed, I've rebuilt it.

I've found the better way of achieving the same results with less code by adding configuration into CsvMapper. I've made some additional research in my spare time. 

The CsvMapper will check fields if those contain commas and wrap any into quotes if needed, which also means explicit checking the aspspName is not needed now.